server/lib/db/index.js

```diff
@@ -32,7 +31,7 @@ class DatabaseConnector {
       user     : process.env.DB_USER,
       password : process.env.DB_PASS,
       database : process.env.DB_NAME,
-      charset  : 'utf8_unicode_ci',
+      charset  : 'UTF8MB4_UNICODE_CI',
```

Comment:
👍 it is important to comment that the charset `UTF8MB4_UNICODE_CI` must be in uppercase.

Reply:
True.  Will do.